content/2_Bronze/Ad_Hoc.problems.json

```diff
@@ -25,6 +25,18 @@
         "kind": "internal"
       }
     },
+    {
```

Comment:
this problem was not originally in this module, so let's not add it here... can you remove this and put it in extraProblems.json instead

Reply:
what's the harm in doing it